apps/console/src/features/users/utils/generate-password.utils.ts

```diff
@@ -0,0 +1,119 @@
+/**
+ * Copyright (c) 2022, WSO2 Inc. (http://www.wso2.com) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+/**
+ * The following function is for generating random numbers in range of [0 - upperlimit)
+ * @param {number} upperLimit - upperlimit (exclusive) for random numbers range
+ * @returns {number}
+ */
+const generateRandomNumbers = (upperLimit: number): number => {
+    return Math.random() * (upperLimit);
+};
+
+/**
+ * The following function is to remove an array of strings from a string
+ * @param {string} str - original string
+ * @param {string[]} ignoredChars - array of strings to be removed
+ * @returns {string} - modified original string by removing ignoredChars
+ */
+const removeIgnoredCharacters = (str: string, ignoredChars: string[]): string => {
+    for (const char of ignoredChars) {
+        str = str.replace(char, "");
+    }
+
+    return str;
+};
+
+/**
+ * The following function is to generate a random password according to the provided options
+ * @param {number} length - length of password
+ * @param {boolean} isAlphabetsLowercaseAllowed - whether to include lowercase alphabets
+ * @param {boolean} isAlphabetsUppercaseAllowed - whether to include uppercase alphabets
+ * @param {boolean} isNumbersAllowed - whether to include numbers
+ * @param {boolean} isSpecialCharactersAllowed - whether to include special characters
+ * @param {number} minAlphabetsLowercase - required minimum number of lowercase alphabets
+ * @param {number} minAlphabetsUppercase - required minimum number of uppercase alphabets
+ * @param {number} minNumbers - required minimum number of numbers
+ * @param {number} minSpecialCharacters - required minimum number of special charaters
+ * @param {string[]} ignoredCharactors - array of charaters to be excluded
+ * @returns {string} - generated password
```

Comment:
Add a full-stop at the end of each comment.

Reply:
Fixed with b017ea7b1473e1a5ff47f46cec10a6963c118668